build.sh

```diff
@@ -99,5 +99,5 @@ else
     cmake $CC_PREFIX -DCMAKE_BUILD_TYPE=Release ..
 fi
 
-make |& tee build.log
+make -j $(nproc) |& tee build.log
```

Comment:
can we make/keep this optional please ? i.e. we can extend build.sh with `-j` option ? (I can take that item)